### PR TITLE
feat: add pocketbase completion spec

### PR DESCRIPTION
**Pull request by ollema**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
This PR will add a completion spec for the PocketBase backend [https://pocketbase.io](https://pocketbase.io)

**What is the current behavior? (You can also link to an open issue here)**
N/A

**What is the new behavior (if this is a feature change)?**
All commands and subcommands are supported now 🙂

**Additional info:**

### CODE REVIEW

**Comment by ollema** [comment]
```./pocketbase --help
PocketBase CLI

Usage:
  pocketbase [command]

Available Commands:
  help        Help about any command
  migrate     Executes DB migration scripts
  serve       Starts the web server (default to 127.0.0.1:8090)

Flags:
      --debug                  enable debug mode, aka. showing more detailed logs
      --dir string             the PocketBase data directory (default "pb_data")
      --encryptionEnv string   the env variable whose value of 32 chars will be used 
                               as encryption key for the app settings (default none)
  -h, --help                   help for pocketbase
  -v, --version                version for pocketbase

Use "pocketbase [command] --help" for more information about a command.```

```

Supported arguments are:
- up                   - runs all available migrations.
- down [number]        - reverts the last [number] applied migrations.
- create name [folder] - creates new migration template file.
- collections [folder] - (Experimental) creates new migration file with the most recent local collections configuration.

Usage:
  pocketbase migrate [flags]

Flags:
      --database string   specify the database connection to use (db or logs) (default "db")
  -h, --help              help for migrate

Global Flags:
      --debug                  enable debug mode, aka. showing more detailed logs
      --dir string             the PocketBase data directory (default "pb_data")
      --encryptionEnv string   the env variable whose value of 32 chars will be used 
                               as encryption key for the app settings (default none)
```

```
Starts the web server (default to 127.0.0.1:8090)

Usage:
  pocketbase serve [flags]

Flags:
  -h, --help              help for serve
      --http string       api HTTP server address (default "127.0.0.1:8090")
      --https string      api HTTPS server address (auto TLS via Let's Encrypt)
                          the incoming --http address traffic also will be redirected to this address
      --origins strings   CORS allowed domain origins list (default [*])

Global Flags:
      --debug                  enable debug mode, aka. showing more detailed logs
      --dir string             the PocketBase data directory (default "pb_data")
      --encryptionEnv string   the env variable whose value of 32 chars will be used 
                               as encryption key for the app settings (default none)
```

File: src/pocketbase.ts [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+      subcommands: [
+        {
+          name: "migrate",
+          description: "Help about the migrate command",
+        },
+        {
+          name: "serve",
+          description: "Help about the serve command",
+        },
+      ],
```
Comment (SeparateRecords) [inline src/pocketbase.ts:17]:
```suggestion
      args: { name: "subcommand", template: "help", isOptional: true },
```

**Review by SeparateRecords — CHANGES_REQUESTED** [review]
Fantastic! Just one quick change then we're good to merge :)

**Review by SeparateRecords — APPROVED** [review]
LGTM! Thank you @ollema 🙂

**Pull request merged**